mautic/mautic
# Fix form set config SDK

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
Fix for https://github.com/mautic/mautic/issues/9951
This https://github.com/mautic/mautic/commit/8b4cd18285b69ecd5f0caefed9d9522ba219ba48 add to generate url param to url to refresh resource on new version of Mautic.
This param produce  issue for Mautic forms 



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Try create and submit form
3. Should works like before
